src/main/java/org/csanchez/jenkins/plugins/kubernetes/pod/retention/Reaper.java

```diff
@@ -146,6 +147,8 @@ private void activate() {
     /**
      * Remove any {@link KubernetesSlave} nodes that reference Pods that don't exist.
      */
+    @SuppressFBWarnings(value = "REC_CATCH_EXCEPTION", justification = "Warning about the Exception x catch at the end. Not sure about the possible"
+                                                                       + "  impact of removing it.")
```

Comment:
This warning is a bit too much, although `catch(Exception e)` could be replaced with `catch (KubernetesAuthException | IOException | RuntimeException e)` in that case.

Reply:
Fixed =)